modules/configuring-firewall.adoc

```diff
@@ -86,11 +86,53 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 |443, 80
 |Required to access Alibaba Cloud services and resources. Review the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/endpoints/endpoints_config.go?spm=a2c4g.11186623.0.0.47875873ciGnC8&file=endpoints_config.go[Alibaba endpoints_config.go file] to determine the exact endpoints to allow for the regions that you use.
 
-|AWS
+.11+|AWS
 |`*.amazonaws.com`
+
+Alternatively, if you choose to not use a wildcard for Amazon Web Services (AWS) APIs, you must allowlist the following URLs:
 |443, 80
 |Required to access AWS services and resources. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
 
+|ec2.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|events.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|iam.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|route53.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|sts.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|tagging.us-east-1.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment. This endpoint is always us-east-1, regardless of the region the cluster is deployed in.
+
+|ec2.<aws_region>.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|elasticloadbalancing.<aws_region>.amazonaws.com
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|servicequotas.<aws region>.amazonaws.com
+|443, 80
+|Required. Used to confirm quotas for deploying the service.
+
+|tagging.<region>.amazonaws.com
+|443, 80
+|Allows the assignment of metadata about AWS resources in the form of tags.
```

Comment:
Leaving this as one comment since it's all the same thing really

```suggestion
|`ec2.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`events.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`iam.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`route53.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`sts.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`tagging.us-east-1.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment. This endpoint is always `us-east-1`, regardless of the region the cluster is deployed in.

|`ec2.<aws_region>.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`elasticloadbalancing.<aws_region>.amazonaws.com`
|443
|Used to install and manage clusters in an AWS environment.

|`servicequotas.<aws region>.amazonaws.com`
|443, 80
|Required. Used to confirm quotas for deploying the service.

|`tagging.<region>.amazonaws.com`
|443, 80
|Allows the assignment of metadata about AWS resources in the form of tags.
```